app/views/pursuits/_pursuit.html.haml

```diff
@@ -0,0 +1,13 @@
+.pursuit{ :id => "#{format_url_name(pursuit.name)}" }
+  .row
+    .col.s12.m7
+      .card
+        .card-image
+          %img(src="jet-skiing.jpg")
+          %span.card-title= "#{ pursuit.name}"
+        .card-content
+          %p= "#{pursuit.description}"
```

Comment:
`%... \#{expression}` can be written without interpolation as `%...= expression`<br>`= "..."` should be rewritten as `...`